src/anomalib/utils/types/task_type.py

```diff
@@ -1,6 +1,6 @@
 """Task type enum."""
 
-# Copyright (C) 2022 Intel Corporation
+# Copyright (C) 2022-2023 Intel Corporation
```

Comment:
```suggestion
# Copyright (C) 2023 Intel Corporation
```

Reply:
as far as I know this is the suggested way when we create the file earlier and modify it later on.

Reply:
Ah didn't know. Let's follow this convention from now on then.